lib/interface/cli/commands/systemRuntimeEnvironments/apply.cmd.js

```diff
@@ -60,6 +69,30 @@ const command = new Command({
         } else {
             await sysRe.update(options, body);
             console.log(`Runtime-Environment: '${options.name}' patched.`);
+            const yamlAccounts = body.accounts;
+            if (yamlAccounts) {
+                if (argv.unassignAccounts) {
+                    const re = await sysRe.get({ ...options, extend: false });
+                    const accounts = _.difference(re.accounts, yamlAccounts);
+                    if (!_.isEmpty(accounts)) {
+                        await sdk.onPrem.runtimeEnvs.account.delete({ name: options.name }, { accounts });
+                        console.log(`Runtime-Environment unassigned from accounts: ${accounts}`);
+                    } else {
+                        console.log('No accounts to unassign');
+                    }
+                }
+
+                if (argv.assignAccounts) {
+                    const re = await sysRe.get({ ...options, extend: false });
+                    const accounts = _.difference(yamlAccounts, re.accounts);
+                    if (!_.isEmpty(accounts)) {
+                        await sdk.onPrem.runtimeEnvs.account.modify({ name: options.name }, { accounts });
+                        console.log(`Runtime-Environment assigned to accounts: ${accounts}`);
+                    } else {
+                        console.log('No accounts to assign');
+                    }
+                }
+            }
```

Comment:
did you check what happen in case of unexpected error? or some account that not exists?

Reply:
good point